client.go

```diff
@@ -465,10 +470,13 @@ func (s *Client) LoopInQuote(ctx context.Context,
 		return nil, ErrSwapAmountTooHigh
 	}
 
+	quote, err := s.Server.GetLoopInQuote(ctx, request.Amount)
+	if err != nil {
+		return nil, err
+	}
+
 	// Calculate swap fee.
```

Comment:
nit: maybe remove the comment or change it since there's nothing calculated any more.

Reply:
Done